kitsune/groups/migrations/0005_create_community_team.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations
+
+
+def create_community_team(apps, schema_editor):
+    """Create the Community Team group with a GroupProfile."""
+    from django.template.defaultfilters import slugify
+    
+    Group = apps.get_model('auth', 'Group')
+    GroupProfile = apps.get_model('groups', 'GroupProfile')
+    
+    group, created = Group.objects.get_or_create(name="Community Team")
+    
+    if not GroupProfile.objects.filter(group=group).exists():
```

Comment:
this can be `if created`

Reply:
nit Why don't you just use get_or_create with some `**defaults` value for creation for both Group and GroupProfile?